### PR TITLE
Added NSFW filter to query

### PR DESCRIPTION
**Pull request by marcdeb1**
Nsfw parameter added to input, for example :
http://127.0.0.1:50005/search?s=lbry&nsfw=true 
returns only nsfw results (see issue #110).
Other filters can easily added to the getFilters function.

### CODE REVIEW

File: server/controllers/lighthouse.js [hunk 1]
```diff
@@ -223,10 +224,30 @@ function getAutoCompleteQuery (query) {
...
-function getFilter (query) {
-  // this is the best place for putting things like filtering on the type of content
-  // Perhaps we can add search param that will filter on how people have categorized / tagged their content
-
+function getFilters (input) {
+    // this is the best place for putting things like filtering on the type of content
+    // Perhaps we can add search param that will filter on how people have categorized / tagged their content
+    var filters = [];
+    if(input.nsfw === "true" || input.nsfw === "false") {
+        const nsfwFilter = {"match": {"value.stream.metadata.nsfw": input.nsfw}}
```
Comment (tiger5226) [inline server/controllers/lighthouse.js]: This worked? I need to double check but I thought it stores the value in elastic search in the same way it is grabbed from Chainquery [[0,1]](https://chainquery.lbry.io/api/sql?query=SELECT%20name,is_nsfw%20FROM%20claim%20WHERE%20name%20=%22awakeningshortfilm%22)

Comment (marcdeb1) [reply server/controllers/lighthouse.js]: It's stored as true/false in Elasticsearch.

**Comment by tiger5226** [comment]
@filipnyquist I reviewed this and tested it locally. All is well. If you have any comments, please note them, otherwise I will merge and deploy this early in the week.

**Comment by marcdeb1** [comment]
Thanks for adding the docs !

**Comment by marcdeb1** [comment]
Thanks for adding the docs !

**Comment by marcdeb1** [comment]
Thanks for adding the docs !

**Comment by marcdeb1** [comment]
Thanks for adding the docs !

**Comment by marcdeb1** [comment]
Thanks for adding the docs !

**Comment by marcdeb1** [comment]
Thanks for adding the docs !

**Comment by tzarebczan** [comment]
@marcdeb1, please shoot us an email to hello@lbry.io with a time estimate for this one too :) 

Thanks for the PR!

**Pull request merged**